index.js

```diff
@@ -40,7 +39,51 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("h1#header")
+
+function handleToggleColor(e) {
+  toggleColor(e.target)
+}
+
+header.addEventListener('click', handleToggleColor)
+
+
 
 /***** Deliverable 2 *****/
+const newPlayerForm = document.querySelector("#new-player-form")
+
+function handlePlayerFormSubmit(e) {
+  e.preventDefault()
+  
+  const number = e.target.number.value
+  const name = e.target.name.value
+  const nickname = e.target.nickname.value
+  const photo = e.target.photo.value
+  
+  const playerObj = {
+    number: number,
+    name: name,
+    nickname: nickname,
+    photo: photo,
+    likes: 999
+  }
+
+  renderPlayer(playerObj) 
+
+  e.target.reset()
+}
+
+newPlayerForm.addEventListener('submit', handlePlayerFormSubmit)
+
+
 
 /***** Deliverable 3 *****/
+playerContainer.addEventListener('click', function (e) {
+  if (e.target.matches(".like-button")) {
+    const likeButton = e.target
+    const playerDiv = likeButton.closest(".player")
+    const likesPtag = playerDiv.querySelector(".likes")
+    const likesCount = parseInt(likesPtag.textContent.replace(' likes',''))
```

Comment:
I like your approach because it's clear what the code is doing; if you want a slightly shorter way though:`parseInt` will actually strip out trailing characters for you (as long as a string starts with a number), so you can also do this:

```js
    const likesCount = parseInt(likesPtag.textContent)
```